packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/saml2assert/setup-app.md

```diff
@@ -1,28 +1,27 @@
-4. Select **Native Application** as the **Application type**, then click **Next**.
-5. Enter a name for your app integration. In the **Sign-in redirect URIs** box, specify the callback location where Okta returns the browser (along with the token).
-6. Click **Save** to create the app.
-7. On the **General** tab that appears, scroll to the **General Settings** section and click **Edit**.
-8. Select **Refresh Token** and **SAML 2.0 Assertion** as the **Allowed grant types**. These selections enable you to exchange an assertion for the access token and also request a refresh token.
+1. Select **Native Application** for **Application type**, then click **Next**.
+1. Enter an **App integration name**.
+1. Enter the **Sign-in redirect URIs** box to redirect the user with their token.
```

Comment:
This doesn't make sense - they enter a box? I suggest starting with "In the" even tho Acrolinx complains. Sometimes it just doesn't work.
```suggestion
1. In the **Sign-in redirect URIs** box, specify the callback location where Okta returns the browser (along with the token). 
```

Reply:
I've changed all five variations around this instruction to similar versions of your suggestion. Used "enter" rather than "verify".